sites/default/sqlconf.php

```diff
@@ -1,26 +1,29 @@
 <?php
 //  LibreEHR
-//  MySQL default Config
-//  Referenced from /library/sqlconf.php.
-
-global $disable_utf8_flag;
-$disable_utf8_flag = false;
+//  MySQL Config
 
 $host	= 'localhost';
 $port	= '3306';
 $login	= 'libreehr';
-$pass	= 'libreehr';
+$pass	= 'Henrietta94';
```

Comment:
@HenriettaWilla  I see your name showing up here. We can't merge this. Please don't commit this file. You might need to send another PR or edit this to not contain this file.

Reply:
Ahaa i seee my fault... https://github.com/LibreHealthIO/lh-ehr/blob/b6a2c04c055ea95be7fb5326cb0a7ecaee586fdc/.gitignore#L6 fixing that asap

Reply:
@Ngai-E Please how am I suppose to ignore the file